2.0/contrib/src/.bashrc

```diff
@@ -1,2 +1,3 @@
 # This will make scl collection binaries work out of box.
+unset BASH_ENV
```

Comment:
I have not seen this in db images or the perl image. Maybe it only happened
to ruby because it was not updated to not source bashrc manually? Will
investigate today.

On 10:41, Wed, Jun 10, 2015 Michal Fojtik <notifications@github.com> wrote:

> In 2.0/contrib/src/.bashrc
> <https://github.com/openshift/sti-ruby/pull/35#discussion_r32098047>:
>
> > @@ -1,2 +1,3 @@
> >  # This will make scl collection binaries work out of box.
> > +unset BASH_ENV
>
> @mnagy <https://github.com/mnagy> @bparees <https://github.com/bparees>
> FYI you have to do this to avoid nasty recursions (@mnagy
> <https://github.com/mnagy>, please update database images)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/sti-ruby/pull/35/files#r32098047>.
>